java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -74,12 +89,12 @@ public static void setValidityBitToOne(ArrowBuf validityBuffer, int index) {
   public static void setValidityBit(ArrowBuf validityBuffer, int index, int value) {
```

Comment:
is this still used?  can it be marked as deprecated?

Reply:
I am afraid not. This will be used in scenarios where the bit to set is unknown a priori. 
For example, in BaseVariableWidthVector#public void set(int index, int isSet, int start, int end, ArrowBuf buffer).